modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -459,4 +459,102 @@ definition {
 		}
 	}
 
+	@description = "LPS-173894 - Verify that it's possible to send an e-mail notification via an Action (Delete)"
+	@priority = 5
+	test CanSendEmailViaActionDelete {
+		property custom.properties = "feature.flag.LPS-173894=true";
+		property portal.acceptance = "true";
+		property test.name.skip.portal.instance = "ObjectNotifications#CanSendEmailViaActionDelete";
+
+		task ("Given: an Email Notification Template") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "Email Body",
+				entryTo = "test@liferay.com",
+				fromAddress = "Liferay",
+				fromName = "Test Test",
+				name = "Notification Template",
+				notificationType = "Email",
+				subject = "Subject");
+
+			PortletEntry.save();
+		}
+
+		task ("And: an Custom Object with an Action (Delete Entry)") {
```

Comment:
Split this into two tasks, one to create the object and another to create the action. We want to be descriptive here so that we can analyze the test quickly by reading the task. Make sure you move the macro to add the entry with the task that creates the object definition.
`And given a custom object with a text field and an object entry`
`And given an Action to send an email notification after deleting an entry`